.github/workflows/ac-ansible-test-sanity.yml

```
@@ -51,23 +51,23 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v5
       with:
-        python-version: 3.11
+        python-version: 3.12
   
     - name: Set up venv
       run: |
         python -m pip install --upgrade pip
         pip install virtualenv
         mkdir venv
-        virtualenv venv/venv-2.16
+        virtualenv venv/venv-2.18
 
     - name: Install dependencies
       run: |
-        source venv/venv-2.16/bin/activate
+        source venv/venv-2.18/bin/activate
         python -m pip install --upgrade pip
-        pip install ansible-core==2.17.6
+        pip install ansible-core==2.18
```

Maybe we could install the latest minor version of 2.18, but agree with Rich that there shouldn't be much of a difference between versions of Ansible 2.18.

```suggestion
        pip install "ansible-core>=2.18,<2.19"
```